content/en/docs/reference/glossary/endpoint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Endpoint
+id: endpoint
+date: 2020-04-23
+full_link: 
+short_description: >
+  Endpoint tracks the IP Addresses of the objects the service send traffic to.
+
+aka:
+tags:
+- fundamental
+---
+ Endpoints tracks the IP Addresses of the objects the service send traffic to.
+
+<!--more-->
+
+Endpoint can be updated automatically through labels and selectors, or you can manually configure your endpoints depending on your use case.
+Multiple Endpoints can be grouped together with {{< glossary_tooltip text="EndPointSlice" term_id="endpoint-slice" >}}.
```

Comment:
nit:
```
Multiple Endpoints can be grouped together with {{< glossary_tooltip text="EndpointSlice" term_id="endpoint-slice" >}}. 
```

Reply:
what do you mean by nit: ? Sorry I am new here!

Reply:
nit comes from meaning 2 in https://en.wiktionary.org/wiki/nitpicking#English

It means that this is a small piece of feedback and it could be OK to overlook it.